CONTRIBUTING.md

```diff
@@ -0,0 +1,230 @@
+# Documentation contribution guide
+
+## For new contributors
+
+"I'm a newcomer, and I'd like to fix an issue."
+
+### Contacting another developer
+
+If you have found a small error or missing piece of information
+and are unable to fix it yourself,
+you can email Tech-talk at <tech-talk@aps.anl.gov>.
+Provide the details of the issue
+and another member of the community should be able to fix it for you.
+You can more information about using Tech-talk can be found at <https://epics.anl.gov/tech-talk/>.
+
+### Editing with GitHub
```

Comment:
I think that this section should probably be removed: I think that using the built-in editors causes more trouble than it's worth, and we should not really encourage that workflow.

(Ask me about a former coworker...)